src/utils/chatService.js

```diff
@@ -35,12 +31,23 @@ export default class ChatService {
   onNewMessage = () => {
     this.event.addListener('newMessage', ((message) => {
       console.log(message);
-    }))
+    }));
   }
 
   onNewListenAddr = () => {
     this.event.addListener('newListenAddr', ((multiaddr) => {
       console.log(multiaddr);
-    }))
+    }));
+  }
+
+  startService = async () => {
+    const xpriv = await bitcoin.getPrivKey();
+    const PeerID = await this.service.start(xpriv);
+    // this.service.dial('/ip4/192.168.0.25/tcp/41381');
```

Comment:
```suggestion
```

This can be removed and instead we could add a `dial = async (address)  => ...` function?